src/main/java/ca/corefacility/bioinformatics/irida/database/changesets/AutomatedSISTRUpdate.java

```diff
@@ -124,21 +125,22 @@ public SISTRFileResult mapRow(ResultSet rs, int rowNum) throws SQLException {
 			} else {
 
 				try {
-					//Read the JSON file from SISTR output
-					@SuppressWarnings("resouce")
+					// Read the JSON file from SISTR output
+					@SuppressWarnings("resource")
```

Comment:
Fixed typo, rest of changes in file are formatting changes applied by the project formatting config.